src/librustc_resolve/resolve_imports.rs

```diff
@@ -653,6 +654,9 @@ impl<'a, 'b:'a> ImportResolver<'a, 'b> {
                     return Some((directive.span,
                                  "cannot glob-import all possible crates".to_string()));
                 }
+                GlobImport { .. } if self.session.features_untracked().extern_absolute_paths => {
```

Comment:
This is something that was accidentally omitted from the lint entirely before, where `use *` is completely forbidden in 2018 and so this starts to lint about that in 2015.

Note that there's still a "bug" in the lint for "remove `extern crate`", I'll file a bug for that shortly